test/record.test.js

```diff
@@ -170,6 +190,30 @@ describe('Record', function() {
             });
         });
 
+        it('saves the record and calls a callback', function(done) {
+            record.set('foo', undefined); // eslint-disable-line no-undefined
+            record.set('baz', 'qux');
+            record.save(function(err, updatedRecord) {
+                expect(err).toBeNull();
+
+                expect(updatedRecord).toBe(record);
+                expect(record.get('foo')).toBeUndefined();
+                expect(record.get('baz')).toEqual('qux');
+
+                expect(table._base.runAction).toHaveBeenCalledWith(
```

Comment:
The invocation of `_base.runAction` doesn't seem particularly important for the caller. A more robust test would involve verifying that an HTTP PUT request is issued. This file sets the precedent, so good on you for finding that and sticking to it! Improving the existing tests is in-scope for our work, but I'll be satisfied if you'd rather address this in a follow-up patch.

Reply:
I updated these tests to use the same mocking we use in all other places.